maya22-control.c

```diff
@@ -192,4 +203,4 @@ int main(int argc, char *argv[])
     hid_exit();
 
     return 0;
-}
+}
```

Comment:
Better to keep the newline in the end of the text file to keep posix line definition in text file happy: https://www.baeldung.com/linux/files-end-with-newlines